### DISCUSSION TITLE
Manual/JobSystemNativeContainer

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#518] [Notes]
# Manual/JobSystemNativeContainer

https://docs.unity3d.com/Manual/JobSystemNativeContainer.html

**Comment by vertxxyz** [comment]
If you're just looking to transfer a single value from a job, the [Collections package](https://docs.unity3d.com/Packages/com.unity.collections@latest) contains [`NativeReference`](https://docs.unity3d.com/Packages/com.unity.collections@latest/index.html?subfolder=/api/Unity.Collections.NativeReference-1.html), which is the functional equivalent of a `NativeArray` of length 1.
